### DISCUSSION TITLE
Visual tracking or tracing of an object's movement over time

### DISCUSSION THREAD

**Discussion by ericisback1** [Stellarium/stellarium#3698] [Q&A]
I'm brand new to this group.  Sorry for any mistakes....

I recently saw some videos here:

https://github.com/Stellarium/stellarium/discussions/3638    <------ Link

that showed the visual "tracing" of an object's movement over time. Would anyone be able to show me how to do this?  I searched the documentation, but I can't seem to find it.  I'm not sure I'm using the correct terminology in my search.

To clarify, I know how to use the Ephemeris tab to show the charted positions of objects:

![stellarium-20240331-233835304](https://github.com/Stellarium/stellarium/assets/165617579/f774b00b-c3c7-4b96-beae-77aa37f128c3)

However, I'm trying to figure out how to actually draw the movement over time as shown in the videos.  Hope that makes sense.

Thanks so much for any help!!

**Comment by gzotti** [comment]
The link you posted is not only the videos, but in fact the whole description of what must be made to produce such videos. 

If you just want to show trails, activate "Show trails" (and play with the sub-options adjacent to it) as seen in Figure 4.11 of the User Guide.

**Reply by ericisback1** [reply]
Thanks for the answer. I am not a programmer, so I was unsure if the code was for, just producing the video, or actually producing the content inside the video. Thanks for the clarification.

Also, when I originally read the guide regarding trails, I did check both boxes; however, it did not seem to create any trails on any of the objects.  If that is what the option is for, I am obviously doing something wrong.

**Reply by gzotti** [reply]
After checking trails, you must start a timelapse and let it run for e.g. a few weeks (simulation time), of course.

**Reply by ericisback1** [reply]
Got it, thanks for the help!
